nixos/modules/virtualisation/containers.nix

```diff
@@ -571,6 +576,16 @@ in
               '';
             };
 
+		    timeoutStartSec = mkOption {
+		      type = types.str;
+		      default = "1min";
+		      description = ''
+		        Time for the container to start. In case of a timeout,
+		        the container processes get killed. See systemd.time(7)
```

Comment:
You can use the `<citerefentry><refentrytitle>man</refentrytitle><manvolnum>1</manvolnum></citerefentry>` incantation here

Reply:
Can you use the suggestion feature to propose your change?

Reply:
Thanks. Done!